installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,134 @@
+[id="manually-creating-iam"]
+= Manually creating IAM
```

Comment:
Manual mode is supported on Azure and GCP as well.

Reply:
Restructuring to include Azure and GCP. Thanks!